ext/rotoscope/rotoscope.h

```diff
@@ -17,6 +17,7 @@
 
 typedef enum {
   RS_OPEN = 1,
+  RS_TRACING,
```

Comment:
We should update the README: https://github.com/Shopify/rotoscope/blob/4a2abd08bc538b9c91dcd9d853afe1e4ad8674ff/README.md#rotoscopestate